babelize/main.py

```diff
@@ -45,27 +47,24 @@ def main():
     subparsers.required = True
     subparsers.dest = 'command'
 
-    link = subparsers.add_parser('link', aliases=['ln'])
-    link.set_defaults(run=do_link)
+    link_s = subparsers.add_parser('link', aliases=['ln'])
+    link_s.set_defaults(run=do_link)
 
-    list = subparsers.add_parser('list', aliases=['ls'])
-    list.set_defaults(run=do_list)
-    list.add_argument(
+    list_s = subparsers.add_parser('list', aliases=['ls'])
+    list_s.set_defaults(run=do_list)
+    list_s.add_argument(
         '--lang',
         '-l',
         help='Only list for selected language',
         action='append'
     )
-    list.add_argument(
+    list_s.add_argument(
         '--depth',
         '-d',
         type=int,
         help='Report at depth, default: %(default)s',
         default=2
     )
-    list.add_argument(
```

Comment:
I noticed verbose wasn't propagating, hence adding it here. Might have just been in the help text?

Reply:
Standard argparse requires to put `-v` in the right place (before the subcommand name). Perhaps we could use something like `ilcli` to avoid this, but also `ilcli` is not really a standard way of providing arguments :-). Not sure, tbh.

Reply:
Ah that makes sense. I think this can be fixed by documentation